From f12c36b2bc2e1db86098c181b88b8003c595e63c Mon Sep 17 00:00:00 2001
From: Renato Golin <renato.golin@linaro.org>
Date: Wed, 20 Jul 2016 09:47:09 +0000
Subject: [PATCH] [docs] fix cmake code-block warning

This will unblock the llvm-sphinx-buildbot, which is currently failing due
to a warning being treated as error.

git-svn-id: https://llvm.org/svn/llvm-project/llvm/trunk@276100 91177308-0d34-0410-b5e6-96231b3b80d8
---
 docs/CMakePrimer.rst | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/docs/CMakePrimer.rst b/docs/CMakePrimer.rst
index 0347790..1e3a09e 100644
--- a/docs/CMakePrimer.rst
+++ b/docs/CMakePrimer.rst
@@ -246,11 +246,11 @@ In general CMake if blocks work the way you'd expect:
 .. code-block:: cmake
 
   if(<condition>)
-    .. do stuff
+    message("do stuff")
   elseif(<condition>)
-    .. do other stuff
+    message("do other stuff")
   else()
-    .. do other other stuff
+    message("do other other stuff")
   endif()
 
 The single most important thing to know about CMake's if blocks coming from a C
@@ -265,7 +265,7 @@ The most common form of the CMake ``foreach`` block is:
 .. code-block:: cmake
 
   foreach(var ...)
-    .. do stuff
+    message("do stuff")
   endforeach()
 
 The variable argument portion of the ``foreach`` block can contain dereferenced
-- 
2.5.5

